src/pages/waiting-room-to-car/waiting-room-to-car.html

```diff
@@ -33,6 +33,13 @@
               <ion-col col-32>
                 <label>Tell me question</label>
               </ion-col>
+              <ion-col col-42>
+                <ion-select value="tell-me-question" okText="Submit" cancelText="Cancel" placeholder="Select a question" (ionChange)="tellMeQuestionChanged($event)">
+                  <ion-option [value]="tellMeQuestion.tellMeQuestionCode" *ngFor="let tellMeQuestion of tellMeQuestions">
+                    {{tellMeQuestion.tellMeQuestionCode}} - {{tellMeQuestion.tellMeQuestionShortName}}
```

Comment:
I presume HTML tags aren't allowed in here, so you couldn't do
`<span class="mes-select-code">{{tellMeQuestion.tellMeQuestionCode}}</span><span class="mes-select-value">{{tellMeQuestion.tellMeQuestionShortName}}</span>` and then style the T1 part in bold?

Reply:
I tried this and it doesn't work, it seems `ion-option` is doing some magic behind the scenes and doesn't directly render the inner HTML.